### PR TITLE
Fixed calculation of overlay position

### PR DESCRIPTION
**Pull request by RebeccaStankus**
https://highfidelity.atlassian.net/browse/DEV-2296

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Can someone from High Fidelity approve this PR?

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21892/HighFidelity-Beta-PR16339-377ffe9.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21892/HighFidelity-Beta-Interface-PR16339-377ffe9.dmg

File: scripts/simplifiedUI/ui/simplifiedUI.js [hunk 1]
```diff
@@ -298,7 +298,7 @@ function updateOutputDeviceMutedOverlay(isMuted) {
         var outputDeviceMutedOverlayBottomY = props.y + overlayDims;
         var inputDeviceMutedOverlayTopY = INPUT_DEVICE_MUTED_MARGIN_TOP_PX;
         if (outputDeviceMutedOverlayBottomY + OUTPUT_DEVICE_MUTED_MARGIN_BOTTOM_PX > inputDeviceMutedOverlayTopY) {
-            overlayDims = 2 * (inputDeviceMutedOverlayTopY - Window.innerHeight / 2 - OUTPUT_DEVICE_MUTED_MARGIN_BOTTOM_PX);
+            overlayDims = 2 * (Window.innerHeight - inputDeviceMutedOverlayTopY / 2 - OUTPUT_DEVICE_MUTED_MARGIN_BOTTOM_PX);
```
Comment (zfox23) [inline scripts/simplifiedUI/ui/simplifiedUI.js]: hmm, I'm not sure that this is right.

the "output device muted" banner should always be in the center of the screen, and the banner should shrink if the y-height of the banner is greater than or equal to the y-height of the screen.

this should have nothing to do with the input device banner at all anymore!

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21892/HighFidelity-Beta-PR16339-377ffe9.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21892/HighFidelity-Quest-Beta-PR16339-377ffe9.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21917/HighFidelity-Beta-PR16339-5831eb2.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21917/HighFidelity-Beta-Interface-PR16339-5831eb2.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21917/HighFidelity-Beta-PR16339-5831eb2.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21917/HighFidelity-Quest-Beta-PR16339-5831eb2.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21917/HighFidelity-Beta-PR16339-5831eb2.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21917/HighFidelity-Beta-Interface-PR16339-5831eb2.exe

**Pull request merged**